cirq/google/common_serializers.py

```diff
@@ -218,3 +218,20 @@ def _near_mod_2(e, t, atol=1e-8):
                                              value_func=lambda _: 0.5),
         ]),
 ]
+"""CZ Serializer"""
+CZ_SERIALIZER = op_serializer.GateOpSerializer(
```

Comment:
I'm always torn calling this CZ.  CPHASE_SERIALIZER or CZ_POW_SERIALIZER since this allows arbitrary phase?  Otherwise this doesn't follow pattern of SYC_SERIALIZER where SYC is a fixed gate.

Reply:
Changed to CZ_POW